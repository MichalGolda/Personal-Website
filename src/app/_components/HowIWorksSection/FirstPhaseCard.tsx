import PhaseCard from "./PhaseCard";
import { PhaseCardColors } from "./constants";

const FirstPhaseCard = () => (
  <PhaseCard
    className="w-full md:w-1/2"
    headline="faza 1"
    title="Określenie oczekiwań klienta"
    color={PhaseCardColors.PHASE_FIRST}
    body="W pierwszej fazie dokładnie analizuję oczekiwania klienta, te informacje stanowią fundament, na którym opiera się proces przygotowania ostatecznego rozwiązania."
    data-aos="fade-right"
    data-aos-duration="500"
  />
);

export default FirstPhaseCard;
