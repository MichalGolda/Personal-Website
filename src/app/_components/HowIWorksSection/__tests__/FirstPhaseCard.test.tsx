import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import FirstPhaseCard from "../FirstPhaseCard";

describe("FirstPhaseCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<FirstPhaseCard />);

    expect(container).toMatchSnapshot();
  });
});
