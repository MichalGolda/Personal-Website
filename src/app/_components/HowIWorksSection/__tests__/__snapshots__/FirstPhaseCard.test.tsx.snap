// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstPhaseCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 border border-solid border-lightGrey p-4 bg-white w-full md:w-1/2"
    data-aos="fade-right"
    data-aos-duration="500"
  >
    <div
      class="flex flex-col"
    >
      <span
        class="before:block flex items-center gap-x-2 text-sm md:text-base font-semibold uppercase before:w-2 before:h-2 before:bg-[#30E6FF] text-[#30E6FF]"
      >
        faza 1
      </span>
      <h4
        class="text-lg md:text-xl lg:text-2xl text-secondary font-bold text-secondary"
      >
        Określenie oczekiwań klienta
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      W pierwszej fazie dokładnie analizuję oczekiwania klienta, te informacje stanowią fundament, na którym opiera się proces przygotowania ostatecznego rozwiązania.
    </p>
  </div>
</div>
`;
